src/frontend/xgboost.cc

```diff
@@ -390,6 +399,17 @@ inline treelite::Model ParseStream(dmlc::Stream* fi) {
 
   // set global bias
   model.param.global_bias = static_cast<float>(mparam_.base_score);
+  std::vector<std::string> exponential_family {
+    "count:poisson", "reg:gamma", "reg:tweedie"
```

Comment:
do we need to do something special for `survival:cox`? or assert that it is not supported?

Reply:
No, it is not necessary, since the label is not in log scale for `survival:cox`.

Reply:
@hcho3 has better understanding of survival model than me.